libfabric.spec

```
@@ -85,69 +99,107 @@ Development files for the libfabric library.
 %endif
 
 %prep
-%autosetup -p1 -n libfabric-%dl_version
+%autosetup -p1 -n %{name}-%{dl_version}
 
 %build
+%if %{__remake_config}
+./autogen.sh
+%endif
 %if (0%{?suse_version} > 0)
 export CFLAGS="%{optflags} -fPIC -pie"
 export CXXFLAGS="%{optflags} -fPIC -pie"
 %endif
-if [ ! -f configure ]; then
-    ./autogen.sh
-fi
 # defaults: with-dlopen can be over-rode:
-%configure %{?_without_dlopen} %{configopts} \
-%ifnarch s390
-        --with-valgrind
+%configure  --disable-static          \
+            --disable-silent-rules    \
+            %{?_without_dlopen}       \
+            --with-valgrind           \
+            --enable-sockets          \
+            --enable-tcp              \
+            --enable-verbs            \
+            --enable-rxm              \
+            --enable-shm              \
+%if %{__build_opx}
+            --enable-opx              \
+%else
+            --disable-opx             \
 %endif
+            --disable-usnic           \
+            --disable-efa             \
+            --disable-dmabuf_peer_mem \
+            --disable-hook_hmem       \
+            --disable-hook_debug      \
+            --disable-trace           \
+            --disable-perf            \
+            --disable-rstream         \
+            --disable-rxd             \
+            --disable-mrail           \
+            --disable-udp             \
+            --disable-psm             \
+            --disable-psm2            \
+            --disable-psm3            \
+            --disable-gni             \
+            --disable-bgq
+
 sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
 sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+%make_build
 
-make %{?_smp_mflags} V=1
 
 %install
 %make_install
-# remove unpackaged files from the buildroot
-rm -f %{buildroot}%{_libdir}/*.la
+find %{buildroot} -name '*.la' -exec rm -f {} ';'
```

FWIW,

```suggestion
find %{buildroot} -name '*.la' -print0 | xargs -r0 rm -f
```

is slightly more efficient as it does not fork/exec `rm` for every file but puts as many files onto a single `rm` command's argument list as will fit, ultimately running `rm` with a lot more arguments and therefore a lot fewer times.

Clearly no need to push a fix for just this, but if you have to push a fix for anything else, this one is a good one to keep in mind for this PR or a future PR.

---

ok thanks, will make a note and push it next time I update the package to a new version.